File: ProblemSet/677A_Vanya and Fence/Python/solution.py
# Date : 28-Sep-2020 By Jatin Sharma

n, h = map(int, input().split())
a = list(map(int, input().split()))
width = 0

for i in range(n):
    if a[i] <= h:
        width +=1
    if a[i] > h:
        width +=2 

print(width)
    

File: ProblemSet/677A_Vanya and Fence/C/solution.c
// Date : 28 - Sep - 2020 By Jatin Sharma

#include <stdio.h>

int main()
{
    int n, h, width = 0;
    scanf("%d%d", &n, &h);
    int a[n];
    for (int i = 0; i < n; i++)
    {
       scanf("%d", &a[i]);
        if (a[i] <= h)
        {
            width++;
        }
        if (a[i] > h)
        {
            width += 2;
        }
    }

    printf("%d", width);
    return 0;
}